### PR TITLE
4886 Make Case.updateCaseNodeData more robust, update MERGED file

### PR DESCRIPTION
**Pull request by rcordovano**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/casemodule/Case.java [hunk 1]
```diff
@@ -2187,15 +2187,29 @@ private void createCaseNodeData(ProgressIndicator progressIndicator) throws Case
     private void updateCaseNodeData(ProgressIndicator progressIndicator) throws CaseActionException {
```
Comment (raman-bt) [inline Core/src/org/sleuthkit/autopsy/casemodule/Case.java]: Could add param for ProgressIndicator  to the header.

File: Core/src/org/sleuthkit/autopsy/casemodule/Case.java [hunk 2]
```diff
@@ -2187,15 +2187,29 @@ private void createCaseNodeData(ProgressIndicator progressIndicator) throws Case
...
-                } catch (CoordinationServiceException | InterruptedException | IOException ex) {
-                    throw new CaseActionException(Bundle.Case_exceptionMessage_couldNotUpdateCaseNodeData(ex.getLocalizedMessage()), ex);
+                } else {
+                    /*
+                     * This is a "legacy" case with no data stored in its case
+                     * directory coordination service node yet, or the node is
+                     * empty due to some error, so create the coordination
+                     * service node data from the case metadata.
+                     */
+                    nodeData = new CaseNodeData(metadata);
```
Comment (raman-bt) [inline Core/src/org/sleuthkit/autopsy/casemodule/Case.java]: Does the LastAccessDate need to be set for nodeData here?

File: Core/src/org/sleuthkit/autopsy/casemodule/Case.java (hunk 1 above)
Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/casemodule/Case.java]: @raman-bt, thanks I definitely need to do that.

File: Core/src/org/sleuthkit/autopsy/casemodule/Case.java (hunk 2 above)
Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/casemodule/Case.java]: @raman-bt, the CaseNodeData constructor that takes a CaseMetaData object to make node data from scratch sets the last accessed date.

**Pull request merged**